torch/_torch_docs.py

```diff
@@ -5110,6 +5110,41 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.heaviside,
+           r"""
+heaviside(input, val, out=None) -> Tensor
+
+Returns a new tensor with each of the elements of :attr:`input`
+computed with Heaviside step function.
```

Comment:
Computes the Heaviside step function for each element in :attr:\`input\`.

Reply:
Addressed!